opentelekomcloud/resource_opentelekomcloud_waf_domain_v1.go

```diff
@@ -45,11 +45,11 @@ func resourceWafDomainV1() *schema.Resource {
 				ForceNew: false,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
-						"front_protocol": {
```

Comment:
How about keeping this name and just modifying the SDK part, so the new version of provider will not break users' original setup.

Reply:
The best way would probably be to deprecate the old ones.
https://www.terraform.io/docs/extend/best-practices/deprecations.html#renaming-a-required-attribute

Reply:
yes, the best way is to deprecate the old ones.

Reply:
I will do a quick fix for this as this blocks others update the golangsdk.

Reply:
Ok. We should still rename the parameters to be consistent with the API though. I can work on that, but it'll probably take a while until I find time to do it. I've created #509 to keep track of that.